SingularityService/src/main/java/com/hubspot/singularity/smtp/SingularitySmtpSender.java

```diff
@@ -30,6 +30,8 @@
 
 import io.dropwizard.lifecycle.Managed;
 
+
+
```

Comment:
super nit-picky, but can we get rid of these extra new lines here and down by the LOG.trace